packages/ember-data/lib/system/model/attributes.js

```diff
@@ -71,13 +73,17 @@ Model.reopenClass({
 
     Example
 
-    ```javascript
-    App.Person = DS.Model.extend({
+    ```app/models/person.js
+    import DS from 'ember-data';
+
+    export default DS.Model.extend({
       firstName: attr(),
       lastName: attr('string'),
       birthday: attr('date')
     });
+    ```
 
+    ```javascript
     var transformedAttributes = Ember.get(App.Person, 'transformedAttributes')
```

Comment:
and in that case I think it should be `Ember.get(Person, 'transformedAttributes')`